File: moneylog_spring/src/main/java/bitc/fullstack503/moneylog_spring/dto/MainListDTO.java
package bitc.fullstack503.moneylog_spring.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.time.LocalDateTime;

@Data
public class MainListDTO {
  @JsonProperty("cate")
  private String cate;

  @JsonProperty ("usee")
  private String usee;

  @JsonProperty ("way")
  private String way;

  @JsonProperty ("amount")
  private int amount;

  @JsonProperty ("transactionDate")
  private LocalDateTime transactionDate;

  @JsonProperty ("type")
  private String type;
}

File: moneylog_app/app/src/main/java/bitc/example/app/dto/MainListDTO.kt
package bitc.example.app.dto

import com.google.gson.annotations.SerializedName

data class MainListDTO(
    @SerializedName("cate")
    var cate: String? = null,
    @SerializedName("usee")
    var usee: String? = null,
    @SerializedName("way")
    var way: String? = null,
    @SerializedName("amount")
    var amount: Int? = null,
    @SerializedName("transactionDate")
    var transactionDate: String? = null,
    @SerializedName("type")
    var type: String? = null
)
